Tests/MathLib/TestLinearSolver.cpp

```diff
@@ -207,10 +207,12 @@ TEST(MathLib, CheckInterface_GaussAlgorithm)
     boost::property_tree::ptree t_solver;
     t_root.put_child("ogs", t_solver);
 
+    using Example = Example1<std::size_t>; 
+
     typedef MathLib::GaussAlgorithm<MathLib::GlobalDenseMatrix<double>, MathLib::DenseVector<double> > LinearSolverType;
-    MathLib::GlobalDenseMatrix<double> A(Example1::dim_eqs, Example1::dim_eqs);
+    MathLib::GlobalDenseMatrix<double> A(Example::dim_eqs, Example::dim_eqs);
```

Comment:
Above line 210, there is a newly introduced alias, Example, to ``Example1<std::size_t>`` as
```
using Example = Example1<std::size_t>; 
```